.circleci/config.yml

```diff
@@ -1,26 +1,28 @@
 version: 2.1
 
 orbs:
-  hmpps: ministryofjustice/hmpps@3.2
+  hmpps: ministryofjustice/hmpps@3.4.0
   slack: circleci/slack@4.4.2
 
+parameters:
+  alerts-slack-channel:
+    type: string
+    default: hmpps_typescript_notifications
+
 executors:
+  builder:
+    docker:
+      - image: 'cimg/node:14.17-browsers'
+    working_directory: ~/app
```

Comment:
move to not use the hmpps/node executor as it means you have to specify version in lots of places and it's not obvious which version of node you are using and theoretically it should match the int-tests and Docker file version

Reply:
agreed, sounds a bit naff to have to specify in multiple places but wondering if there is a better way :thinking:

Reply:
if you declared `14.17-browsers` as a parameter like `alerts-slack-channel` then you could just reference the parameter in all usages in the file and then wouldn't have the issue?

Reply:
I did that originally and undid it 😄  

Will try and tidy up in a follow up